product/quickstart.mdx

```diff
@@ -1,62 +1,75 @@
 ---
-title: "Quickstart"
-description: "Not sure where to start? We've put together a few articles for you to quickly get familiar with the concepts and features of Makeswift."
-icon: "bolt" iconType: "solid"
+title: "Creating your site"
+description: "Learn the basics of building with Makeswift including components, theming, responsive design, and publishing."
+icon: "circle-plus"
 ---
 
-<CardGroup cols={3}>
-  <Card title="Concepts" icon="lightbulb" href="/product/concepts">
-    Learn terminology to help you easily navigate Makeswift
-  </Card>
-  <Card
-    title="Builder basics"
-    icon="hammer-brush"
-    iconType="solid"
-    href="/product/builder-basics"
-  >
-    Learn how to interact with instances of a component
-  </Card>
-  <Card title="Layout" icon="table-layout" href="/product/layouts">
-    Drag and drop elements on your page in the Makeswift builder
-  </Card>
-  <Card
-    title="Components"
-    icon="Cube"
-    iconType="solid"
-    href="/product/components"
-  >
-    Leverage the power of React components to build your site to spec
-  </Card>
-  <Card title="Theming" icon="palette" href="/product/theming">
-    Standardize the theme of your page while optimizing for performance
-  </Card>
-  <Card title="Files" icon="folder-open" href="/product/files">
-    Manage files and images for your site
-  </Card>
-  <Card
-    title="Responsive design"
-    icon="laptop-mobile"
-    iconType="solid"
-    href="/product/responsive-design"
-  >
-    Design your website to be responsive across all devices
-  </Card>
-  <Card title="SEO" icon="magnifying-glass-chart" href="/product/SEO">
-    Optimize SEO for your site more easily than ever
-  </Card>
-  <Card
-    title="Localization"
-    icon="map-location-dot"
-    href="/product/localization"
-  >
-    Manage content across different locales
-  </Card>
-  <Card
-    title="Publishing"
-    icon="rocket-launch"
-    iconType="solid"
-    href="/product/publishing"
-  >
-    Push changes to your live domain instantly
-  </Card>
-</CardGroup>
+If you already have a site and want to create a new one, click the **Create new site** button under the site list dropdown. If this is your first time using Makeswift, [create a free account](https://app.makeswift.com/signup). You'll then be taken to the page for creating a new site.
+
+You have two options to get started: **Start with the builder** or **Integrate with Next.js**.
+
+<Info>
+  Starting with the builder utilizes Makeswift's [default
+  host](/product/site/hosting#default-host) while Integrating with Next.js will
+  prompt you to utilize your own [custom
+  host](/product/site/hosting#custom-host).
```

Comment:
For some reason this link isn't scrolling all the way down. Might be an issue with Mintlify 😞

Reply:
I think it's actually just the fact that there's not enough content to scroll the heading up to the top. I think this would happen regardless unfortunately. Solution would be to add more content below it haha